docs/source/tensor_view.rst

```diff
@@ -0,0 +1,61 @@
+.. currentmodule:: torch
+
+.. _tensor-view-doc:
+
+Tensor Views
+=================
+
+PyTorch has decoupled notion of tensor and storage. Storage is the actual physical data, while tensor
+defines the logical interpretation of storage. There's always a pair of tensor-storage, and
+multiple tensors might share the same storage.
+
+.. image:: _static/img/tensor-storage.png
+
+PyTorch also provides views of tensors, instead of creating a new tensor with new storage,
+it just returns a tensor which is a different view on the underlying storage. Supporting views
```

Comment:
you should probably just include a simple program that demonstrates a view -- that's much easier to understand than an abstract description.